source/android/adaptivecards/src/main/java/com/microsoft/adaptivecards/renderer/readonly/ColumnRenderer.java

```diff
@@ -59,30 +59,34 @@ public View render(
             Vector<IInputHandler> inputActionHandlerList,
             HostConfig hostConfig)
     {
+        GridLayout.LayoutParams layoutParams = new GridLayout.LayoutParams();
         BaseCardElementVector baseCardElementVector = column.GetItems();
         View returnedView = CardRendererRegistration.getInstance().render(context, fragmentManager, null, column, baseCardElementVector, inputActionHandlerList, hostConfig);
-        String columnSize = column.GetSize().toLowerCase(Locale.getDefault());
+        String columnSize = column.GetWidth().toLowerCase(Locale.getDefault());
         if (TextUtils.isEmpty(columnSize) || columnSize.equals(g_columnSizeAuto))
         {
-            GridLayout.LayoutParams param = new GridLayout.LayoutParams(GridLayout.spec(0), GridLayout.spec(index));
-            returnedView.setLayoutParams(param);
+            //GridLayout.LayoutParams layoutParams = new GridLayout.LayoutParams(GridLayout.spec(0), GridLayout.spec(index));
```

Comment:
>//GridLayout.LayoutParams layoutParams = new GridLayout.LayoutParams(GridLayout.spec(0), GridLayout.spec(index)); [](start = 12, length = 113)

Do you plan to reenable these lines in the future?

Reply:
Yeah, not sure if I plan to go with the new or old.  Neither solves the restaurant.json sample scenario.